code/modules/mob/living/carbon/xenomorph/castes/crusher/abilities_crusher.dm

```diff
@@ -97,11 +97,11 @@
 	X.face_atom(L) //Face towards the target so we don't look silly
 
 	var/facing = get_dir(X, L)
-	var/toss_distance = rand(3,5)
+	var/toss_distance = X.xeno_caste.crest_toss_distance //rand(3,5)//no more random.
```

Comment:
How about splitting the difference to a distance of 4 instead of nerfing it to 3?

Reply:
Actually it would be badass if it increased with age like stomp damage: 3, 4, 5, 5

Reply:
put on 3 for test i dunno if 3 goes for like 3 tiles counting crusher or 3 after crusher

Reply:
Pretty sure it is 3 tiles after getting put behind if not harm intent, also unless you changed it that distance goes into the damage calc too.

Reply:
yeah i'll do some test, gonna compile and test changing this.